### DISCUSSION TITLE
Page static only if not authenticated

### DISCUSSION THREAD

**Discussion by itsfolf** [vercel/next.js#17207] [Help] [answered]
Hello there, I'm new to next.js and would like to know if what I'm thinking of is possible. Basically, I want my index page to be a static landing page, benefiting from static pre-rendering and optimization, but I also want that same page to, when accessed by an authenticated user, display something completely different (such as a dashboard). I know it would probably be simpler to just use a /app page or something of sorts, but I'd like to know if this is possible and how it could be done.

**Comment by leerob** [comment, marked as answer]
You can use the same pattern that Vercel uses, which is redirecting from index (`/`) to your dashboard (`/dashboard`) if the user is logged in. You could check for the user inside React, or set a cookie when the user logs in and read that in the `<head />`. For example:

```js
<Head>
    <script
        dangerouslySetInnerHTML={{
        __html: `
        if (document.cookie && document.cookie.includes('auth')) {
        window.location.href = "/dashboard"
        }
    `
        }}
    />
</Head>
```

**Reply by rahul3103** [reply]
But, this same problem I am facing and this is not the answer. He doesn't want to change his route. I have the same use case.
@leerob

**Reply by leerob** [reply]
Replied in the other discussion: https://github.com/vercel/next.js/discussions/17248#discussioncomment-82100
